federation-services/latest/support/index.md

```diff
@@ -2,15 +2,14 @@
 title: Supported platforms
 ---
 
-The following are the supported platforms for Alfresco Federation Services 1.0:
+The following are the supported platforms for Alfresco Federation Services 1.1:
 
 ## Alfresco Content Services
 
 | Version | Supported | Notes |
 | ------- | --------- | ----- |
-| Content Services 6.2 | | |
-| Content Services 6.1 | | |
-| Content Services 6.0 | | |
+| Content Services 7.0 | | |
```

Comment:
ACS Compatibility is 6.0.x, 6.1.x, 6.2.x.  This version does not support ACS 7.x.

Reply:
Made changes as per Andy's request cheers

Reply:
The file does not seem to have changed?

Reply:
this is an outdated file by the looks, it seems like its all good at my end here. Not sure what outdated means.